From 0f6619aee86f11cee0c5063777c4febdf18cb28b Mon Sep 17 00:00:00 2001
From: Dan Carpenter <dan.carpenter@oracle.com>
Date: Wed, 7 Jul 2021 13:15:40 +0300
Subject: [PATCH] ksmbd: delete some stray tabs
Git-commit: 0f6619aee86f11cee0c5063777c4febdf18cb28b
Patch-mainline: v5.15-rc1
References: bsn#12
Modified-by-SEL: No


These lines are intended one tab too far.

Signed-off-by: Dan Carpenter <dan.carpenter@oracle.com>
Signed-off-by: Namjae Jeon <namjae.jeon@samsung.com>
Signed-off-by: Steve French <stfrench@microsoft.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 fs/ksmbd/smb2pdu.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/fs/ksmbd/smb2pdu.c b/fs/ksmbd/smb2pdu.c
index 18e275abc68f..d81768431249 100644
--- a/fs/ksmbd/smb2pdu.c
+++ b/fs/ksmbd/smb2pdu.c
@@ -6699,7 +6699,7 @@ int smb2_lock(struct ksmbd_work *work)
 			    cmp_lock->start < smb_lock->end) {
 				pr_err("previous lock conflict with zero byte lock range\n");
 				rsp->hdr.Status = STATUS_LOCK_NOT_GRANTED;
-					goto out;
+				goto out;
 			}
 
 			if (smb_lock->zero_len && !cmp_lock->zero_len &&
@@ -6707,7 +6707,7 @@ int smb2_lock(struct ksmbd_work *work)
 			    smb_lock->start < cmp_lock->end) {
 				pr_err("current lock conflict with zero byte lock range\n");
 				rsp->hdr.Status = STATUS_LOCK_NOT_GRANTED;
-					goto out;
+				goto out;
 			}
 
 			if (((cmp_lock->start <= smb_lock->start &&
-- 
2.33.0

